sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -220,12 +220,13 @@ def get_all_options(self, drop_default=False):
     if unknown_args and unknown_args[0] != '':
       logging.info("Parsing unknown args: %s", unknown_args)
       for arg in unknown_args:
-        if arg.startswith('--'):
+        # https://issues.apache.org/jira/browse/BEAM-5442
+        if arg.startswith('--') and not arg.startswith('--beam_plugins'):
```

Comment:
Will do a follow-up because this just cures a symptom. We need an extra check to avoid registering the same option twice.